src/mpid/ch4/netmod/ofi/ofi_init.c

```diff
@@ -904,24 +908,29 @@ int MPIDI_OFI_mpi_finalize_hook(void)
     MPIDIU_map_destroy(MPIDI_OFI_global.huge_recv_counters);
 
     if (MPIDI_OFI_ENABLE_AM) {
```

Comment:
Is there any case where AM isn’t enabled at this point? Can we think about removing this capability set?

Reply:
I think we should (assume AM is always enabled). The library won't be functional if we can't even enable AM.

Reply:
I assume this will happen in another PR...

Reply:
Yes, let's do that separately.